spec/support/holdings_tables.rb

```diff
@@ -9,11 +9,11 @@
     hf_db.recreate_tables!
   end
 
-  before(:each) do
-    db[:oclc_concordance].truncate
-    db[:cluster_ocns].truncate
-    Services.hathifiles_table.truncate
-    Services.holdings_table.truncate
+  around(:each) do |example|
```

Comment:
Recommended testing method for Sequel https://github.com/jeremyevans/sequel/blob/master/doc/testing.rdoc -- significantly faster than truncation; avoids strange issues with cached queries after a truncate.

Reply:
I'm glad you latched onto this discussion, I'm filing it away for future reference.